packages/libs/react-components/src/components/Typography/GradientText/GradientText.tsx

```diff
@@ -0,0 +1,11 @@
+import { GradientText as StyledText } from '../styles';
+
+import React, { FC } from 'react';
+
+export interface IGradientTextProps {
+  children: React.ReactNode;
+}
+
+export const GradientText: FC<IGradientTextProps> = ({ children }) => {
+  return <StyledText>{children}</StyledText>;
+};
```

Comment:
You could simplify this further by just exporting the Styled component rather than rewriting it as a function component

Reply:
I was thinking that we might add a font weight property to this component, if you guys don't think that would be necessary that we can indeed just export the styled-component?

Reply:
even with a fontWeight property I don't think writing it as a function component would be necessary since stitches will type the variant for you